content/tooling/wrangler/configuration/environments.md

```diff
@@ -87,7 +87,7 @@ $ wrangler publish
 
 #### Introducing Environments
 
-This `wrangler.toml` adds two environments to the base case.
+This `wrangler.toml` adds two environments to the base case. Note that you must provide a route/routes key for each environment.
```

Comment:
do we need to say this? Won't wrangler throw an error if they do? And they don't need to provide a route/routes in each environment if they are just using workers.dev right?

Reply:
wrangler will throw an error; i think i would maybe add to this "if you are deploying to a zone", because the message is that route/s don't inherit like zone id and workers_dev keys do

Reply:
Your change is fine with me 👍

Reply:
redoing this suggestion as i smashed it with a force push
```suggestion
This `wrangler.toml` adds two environments to the base case. Note that you must provide a route/routes key for each environment if you are deploying to a custom domain.
```